_layouts/post.html

```diff
@@ -75,32 +75,26 @@ <h1 class="display-6 mt-4 text-center">{{ page.title }}</h1>
   </div>
   </div>
 
-  <!-- LIGHTBOX PLUGIN -->
-  <script>
-    let postImages = document.querySelectorAll("article.blog-article img");
+<!-- LIGHTBOX PLUGIN -->
+<script src="{{ "/plugins/lightbox/js/lightbox-plus-jquery.min.js" | relative_path }}"></script>
+<link href="{{ "/plugins/lightbox/css/lightbox.min.css" | relative_path }}" rel="stylesheet" />
+<script src="{{ "/plugins/lightbox/lightbox-decorator.js" | relative_path }}"></script>
```

Comment:
Lines 78-81 have nothing to do with detecting read status. This is just to fix a bug introduced by a prior PR.